react/src/components/HeaderMenu/HeaderMenuContainer.tsx

```diff
@@ -106,39 +107,39 @@ const HeaderMenuContainer = () => {
             break;
           }
           case QrResolverResultKind.NOT_BOXTRIBUTE_QR: {
-            notificationVar({
+            createToast({
```

Comment:
I think here and in lines 118, 126, 134 we could use the `triggerError` utility rather to also have the error logged to Sentry.